Bluepill-cli/BPInstanceTests/BluepillTests.m

```diff
@@ -344,6 +344,137 @@ - (void)testRunWithPassingTestsSet {
     XCTAssert(exitCode == BPExitStatusTestsAllPassed);
 }
 
+- (void)testReuseSimulator {
+    //[BPUtils quietMode:NO];
```

Comment:
do you mean remove "//[BPUtils quietMode:NO];", but I find it very useful to uncomment it during test & debug. If we don't like commented code, I would like to uncomment and disable quite mode.